kahuna/public/js/image/service.js

```diff
@@ -7,7 +7,8 @@ export const imageService = angular.module('gr.image.service', ['kahuna.edits.se
 imageService.factory('imageService', ['editsService', function(editsService) {
     function forImage(image) {
         return {
-            usageRights: usageRights(image)
+            usageRights: usageRights(image),
+            states: getStates(image)
```

Comment:
Not sure states sounds right here i.e. It's in the hasCrops state?
I can't think of anything better though :bike:...